class UsuarioNoEncontradoError(Exception):
    """Se lanza cuando se intenta acceder, modificar o eliminar un usuario que no existe."""
    pass